src/Infrastructure/OrchestratorService.cs

```diff
@@ -46,6 +46,17 @@ public async Task FetchMapAndSaveEmployeeData()
 
         var phoneNumberUtil = PhoneNumberUtil.GetInstance();
 
+        foreach (var bemanning in bemanningEntries.Where(employee => !IsActiveEmployee(employee)))
+        {
+            // Remove potential employees that shouldn't have been added.
+            // This should normally not happen, but might happen in cases where
+            // StartDate in bemanning wasn't set properly when orchestrating.
+            _logger.LogInformation(
+                "Deleting employee with email {BemanningEmail} from database, since they haven't started yet",
+                bemanning.Email);
+            await _employeesRepository.EnsureEmployeeIsDeleted(bemanning.Email);

```

Comment:
This possibly returns a blobUrl, at the end of the function a list of blobs of previous employees is cleaned. Could be nice to add returned images to that list.